tests/functional/aws-node-sdk/test/support/config.js

```diff
@@ -58,11 +85,11 @@ function _getAwsConfig(profile, config) {
     return awsConfig;
 }
 
-function getConfig(profile = 'default', config = {}) {
-    const fn = process.env.AWS_ON_AIR && process.env.AWS_ON_AIR === 'true'
-        ? _getAwsConfig : _getMemConfig;
-
-    return fn.apply(this, [profile, config]);
+function getConfig(profile, config) {
+    if (process.env.AWS_ON_AIR) {
```

Comment:
Are you sure about that ? This one is essential `process.env.AWS_ON_AIR === 'true' as `AWS_ON_AIR = false` will consider it as `AWS_ON_AIR` in your case. An env var is always a string